app/game/middleware/server-pipe.js

```diff
@@ -40,17 +40,17 @@ function getAction(message) {
         case MESSAGE_NAME.roundCountdownChanged:
             return RoundActionsCreator.updateRemaining(message.roundCountdown);
         default:
-            warn('Unknown message from server');
+            return null;
     }
-
-    return null;
 }
 
 function handleServerMessage(message, dispatch) {
```

Comment:
I guess, that  shared `handleServerMessage` gives low profit and increase list of module dependencies